src/types.ts

```diff
@@ -0,0 +1,49 @@
+export interface UserProps {
```

Comment:
Почему это называется props, если это интерфейс которые описывает объект?
```suggestion
export interface UserInterface {
```
```suggestion
export interface User {
```